src/etwlogger_driver.c

```diff
@@ -169,6 +169,23 @@ static void perform_EventWriteLogErrorEvent(const char* content, const char* fil
 #endif
 }
 
+static void perform_EventWriteLogWarningEvent(const char* content, const char* file, const SYSTEMTIME* t, const char* func, int line)
+{
+    (void)t;
+    TraceLoggingWrite(g_hMyComponentProvider,
+        "LogError",
+        TraceLoggingLevel(TRACE_LEVEL_WARNING),
+        TraceLoggingString(file, "file"),
+        TraceLoggingString(func, "func"),
+        TraceLoggingInt32(line, "line"),
+        TraceLoggingString(content, "content")
```

Comment:
>TraceLoggingString(content, "content") [](start = 8, length = 38)

Can we make "content" the first item (here and everywhere)? It makes it easier to view in perfview if you need to collect logs locally (because that only shows the first field).

Reply:
Sure

---
In reply to: [425321988](https://github.com/Azure/azure-c-logging/pull/3#discussion_r425321988) [](ancestors = 425321988)